lib/data-factory.js

```diff
@@ -45,4 +45,4 @@ DataFactory.quad = function (subject, predicate, object, graph) {
 
 DataFactory.defaultGraphInstance = new DefaultGraph()
 
-module.exports = DataFactory
+export default DataFactory
```

Comment:
Default export should be avoided with ES modules. At least for non-internal modules

It would be better to have this as `export BlankNode from './blank-node'`